console/console-init/ui/src/modules/address-space/components/AddressSpaceConfiguration/AddressSpaceConfiguration.tsx

```diff
@@ -175,6 +180,16 @@ export const AddressSpaceConfiguration: React.FC<IAddressSpaceConfigurationProps
                 value={authenticationService}
               />
             </FormGroup>
+
+            <FormGroup fieldId="customize-endpoint">
+              <br />
+              <Switch
```

Comment:
Any reason for not using the generic switch component here?

Reply:
This branch doesn't contain IOT components.